Makefile

```diff
@@ -325,6 +333,7 @@ $(STATUS_CLIENT_DMG): nim_status_client $(DMG_TOOL)
 	mkdir -p $(MACOS_OUTER_BUNDLE)/Contents/Resources
 	cp Info.plist $(MACOS_OUTER_BUNDLE)/Contents/
 	cp bin/nim_status_client $(MACOS_OUTER_BUNDLE)/Contents/MacOS/
+	cp dictionaries/* $(MACOS_OUTER_BUNDLE)/Contents/MacOS/dictionaries
```

Comment:
Should there be a corresponding operation for `$(STATUS_CLIENT_EXE)`?

Reply:
now its just for mac os but in future yes.